handout/directives/ng_style_directive.md

```diff
@@ -40,7 +40,7 @@ We can remove the style properties out of the template into the component as a p
 export class StyleExampleComponent {
   borderStyle: string = '1px solid black';
 
-  // Must be JSON valid.
+  // Properties must be quoted.
```

Comment:
It just has to be enumerable object. Quotes are only required if it's a plain 'ol object and the CSS property contains a dash, which would not be valid, e.g. `border-radius`, in which case you may use camel case property values, e.g. `borderRadius`.

Here's some different ways to use ngStyle (e.g. using a class with properties defined programatically)

https://plnkr.co/edit/bDTs3s?p=preview